test/WireMock.Net.Tests/WireMockServerTests.cs

```diff
@@ -133,6 +136,47 @@ public async Task WireMockServer_WithCorsPolicyOptions_Should_Work_Correct()
 }
 #endif
 
+#if NET461 || NETSTANDARD1_3 || NETCOREAPP3_1 || NET5_0 || NET6_0 || NET7_0
+    [Fact]
+    public async Task WireMockServer_WithRequiredClientCertificates_Should_Work_Correct()
```

Comment:
Can you place this unit test in a separate file?

Like `WireMockServerTests.ClientCertificate.cs`